osmosis-1/osmosis-1.assetlist.json

```diff
@@ -8065,6 +8065,49 @@
         "osmosis-info",
         "OSMO:969"
       ]
-    }
+    },
+    {
+      "description": "The native token of OmniFlix Hub chain",
+      "denom_units": [
+        {
+          "denom": "ibc/CEE970BB3D26F4B907097B6B660489F13F3B0DA765B83CC7D9A0BC0CE220FA6F",
+          "exponent": 0,
+          "aliases": [
+            "uflix"
+          ]
+        },
+        {
+          "denom": "flix",
+          "exponent": 6
+        }
+      ],
+      "base": "ibc/CEE970BB3D26F4B907097B6B660489F13F3B0DA765B83CC7D9A0BC0CE220FA6F",
+      "name": "OmniFlix Network",
+      "display": "flix",
+      "symbol": "FLIX",
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "omniflixhub",
+            "base_denom": "uflix",
+            "channel_id": "channel-1"
+          },
+          "chain": {
+            "channel-id": "channel-199",
+            "path": "transfer/channel-199/uflix"
```

Comment:
Can we please register this ibc connection to the chain registry? 
https://github.com/cosmos/chain-registry/tree/master/_IBC

Reply:
added to the chain-registry
https://github.com/cosmos/chain-registry/pull/1890